File: 240214/함수를 이용한 합과 소수 판별/use-functions-to-determine-sums-and-decimals.py
a, b = map(int, input().split())

def is_prime(n):
    if n == 1:
        return False
    for i in range(2, n):
        if n % i == 0:
            return False
    return True

def even(n):
    return (n // 10 + n % 10) % 2 == 0

cnt = 0
for i in range(a, b+1):
    if is_prime(i) and even(i):
        cnt += 1

print(cnt)

File: 240806/함수를 이용한 합과 소수 판별/use-functions-to-determine-sums-and-decimals.js
const fs = require("fs");
let [a, b] = fs.readFileSync(0).toString().trim().split(" ").map(Number);
let cnt = 0;

function isPrimeNumber(n) {
    for (let i = 2; i < n; i++) {
        if (n % i === 0) return false;
    }
    return true;
}

function even(n) {
    let sum_val = 0
    if (n === 100) return false;
    sum_val = parseInt(n / 10) + n % 10;
    
    return sum_val % 2 === 0;
}


for (let i = a; i <= b; i++) {
    if (isPrimeNumber(i) && even(i)) cnt++;
}

console.log(cnt);